src/StorageWriter/ClaimController.ts

```diff
@@ -56,7 +56,7 @@ export class ClaimController {
     return { claim }
   }
 
-  private readonly uploadClaim = (claim: SignedVerifiableClaim) => this.ipfs.addText(JSON.stringify(claim))
+  private readonly uploadClaim = (claim: SignedVerifiableClaim) => this.ipfs.addText()(JSON.stringify(claim))
```

Comment:
tried to do asyncPipe like so, but the types are complaining for some reason, might have to do with asyncPipe types that need to be adjusted. Regardless It can be a refactor for another time.

```ts
  private readonly configureUpload = ({ timeout = minutesToMiliseconds(10) } = {}) => this.ipfs.addText({ timeout })

  private readonly uploadClaim = (claim: SignedVerifiableClaim) => asyncPipe(
    JSON.stringify,
    this.ipfs.addText()
  )
```